main.go

```diff
@@ -448,3 +403,154 @@ func Main(args *CLIArgs) error {
 
 	return nil
 }
+
+func generateOutput(packages string, outputFormat string, outputType string, mainMods map[string]struct{}, mainLibPkgs []string, mainCmdPkgs []string,
+	modNames []string, modLicenses map[string]map[detectlicense.License]struct{}, modInfos map[string]*golist.Module, goVersion string) (*bytes.Buffer, error) {
```

Comment:
These lines still run over the github window size - makes it a bit harder to review. I don't think this is a blocking problem but we should be aware of it and maybe seek out an autoformatting system to use in the future. Everything else LGTM.

Reply:
Worth noting I'm not comfortable on the language/project so I would still wait for @LukeShu to review before merging if he's available.

Reply:
> These lines still run over the github window size - makes it a bit harder to review. I don't think this is a blocking problem but we should be aware of it and maybe seek out an autoformatting system to use in the future. Everything else LGTM.

Yes, github displays like 95 characters. I went with what I could see on my display but I agree it depends how you look at the PR.

@LukeShu any suggestions about this? Is there a good reference we can use company-wide for Go code style?
Found some guidelines Uber published: https://github.com/uber-go/guide/blob/master/style.md#style but it's more than just style. Maybe we want to adjust this to how we work.

BTW, that guide suggests a limit of 99 characters per line.